models/safe/arbitrum/safe_arbitrum_safes.sql

```diff
@@ -0,0 +1,51 @@
+{{ 
+    config(
+        materialized='incremental',
+        alias='safes',
+        partition_by = ['block_date'],
+        unique_key = ['block_date', 'address'],
+        on_schema_change='fail',
+        file_format ='delta',
+        incremental_strategy='merge',
+        post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                    "project",
+                                    "safe",
+                                    \'["tschubotz"]\') }}'
+    ) 
+}}
+
+select
+    'arbitrum' as blockchain,
+    et.from as address,
+    case 
+        when et.to = '0x8942595a2dc5181df0465af0d7be08c8f23c93af' then '0.1.0'
```

Comment:
Yes, confirming they are consistent across chains. 

Or rather: if they have the same address, then they are for sure the version indicated. 

Now for this specific one, it doesn't exist on Aribtrum (https://arbiscan.io/address/0x8942595a2dc5181df0465af0d7be08c8f23c93af). Reason is we never deployed it. Theoretically it could be deployed though. It's unlikely for the specific one since it's a very old Safe version, but for others that's more realistic.

I figured I rather keep always all addresses in there to keep the queries consistent. And also to be ready in case a missing one finally gets deployed.

Reply:
got it! thank you for quick clarification. really clean PRs by the way, much appreciated 👍